app/controllers/loans_controller.rb

```diff
@@ -0,0 +1,4 @@
+class LoansController < ApplicationController
+  def new
```

Comment:
Please, let's change `new` for create

The `new` action is used to [serve forms views](https://guides.rubyonrails.org/action_controller_overview.html#methods-and-actions)